synapse/models/biz.py

```diff
@@ -141,28 +150,59 @@ def getModelDefs(self):
                     ('purchase', ('econ:purchase', {}), {
                         'doc': 'Records a purchase resulting from the deal.',
                     }),
-                    # TODO
-                    # -(deal:comms)>
-                    # -(buyer:contacts)>
-                    # -(seller:contacts)>
                 )),
                 ('biz:bundle', {}, (
                     ('count', ('int', {}), {
-                        'doc': 'The number of instances of the product included in the bundle.',
+                        'doc': 'The number of instances of the product or service included in the bundle.',
                     }),
                     ('price', ('econ:price', {}), {
                         'doc': 'The price of the bundle.',
                     }),
                     ('product', ('biz:product', {}), {
                         'doc': 'The product included in the bundle.',
                     }),
+                    ('service', ('biz:service', {}), {
+                        'doc': 'The service included in the bundle.',
+                    }),
                     ('deal', ('biz:deal', {}), {
-                        'doc': 'The deal which includes this bundle.',
+                        'deprecated': True,
+                        'doc': 'Deprecated. Please use econ:receipt:item for instances of bundles being sold.',
                     }),
                     ('purchase', ('econ:purchase', {}), {
-                        'doc': 'The purchase which includes this bundle.',
+                        'deprecated': True,
+                        'doc': 'Deprecated. Please use econ:receipt:item for instances of bundles being sold.',
                     }),
                 )),
+                ('biz:listing', {}, (
```

Comment:
Maybe a :desc prop to capture product/service descriptions provided by the seller?

Reply:
Another good idea for a ticket.

Reply:
Maybe roll them up into one ticket for additional suggested fields. Model PR reviews are mostly for show stoppers / necessary clarifications rather than suggesting new fields. :+1: